chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -192,6 +192,21 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             endChat(props, chatSDK, setAdapter, setWebChatStyles, dispatch, adapter, false, false, false);
         });
 
+        // Close popout window
+        BroadcastService.getMessageByEventName("ClosePopoutWindow").subscribe(() => {
```

Comment:
Please add this to BroadcastEvent object and log it in Telemetry.md

Reply:
Please do so for all events in useEffect that is not in BroadcastEvent yet

Reply:
done